ariadne/core/src/Ariadne/UIConfig.hs

```diff
@@ -97,6 +107,41 @@ deleteRetypeMkMessage :: Format Text (delItem -> Text) -> delItem -> Text
 deleteRetypeMkMessage itemTypeFormat itemType = sformat
     ("Type " % itemTypeFormat % " name to confirm deletion") itemType
 
+rmUnkownKeysHeaderMessage :: Text
+rmUnkownKeysHeaderMessage = "Remove unknown keys"
+
+rmUnknownKeysSureMessage :: Text
+rmUnknownKeysSureMessage = "Make sure that you don't need keys from keyfile \
+    \(It is not a backup, for example). Those keys will be deleted."
+
+rmUnknownKeysIntroMessage :: Text
+rmUnknownKeysIntroMessage= "There are keys, that don't \
```

Comment:
Comma should be here, I believe.

Reply:
Actually, could you check these messages in Grammarly?

Reply:
ok